<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>surfacing_Dialog</class>
 <widget class="QDialog" name="surfacing_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surfacing_Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="depth_label">
     <property name="text">
      <string>depth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="depth_lineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="length_label">
     <property name="text">
      <string>length</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="length_lineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="feed_label">
     <property name="text">
      <string>feed rate
[units/min]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="feed_lineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="rpm_label">
     <property name="text">
      <string>rpm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="rpm_lineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="unit_label">
     <property name="text">
      <string>units</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="unit_comboBox"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="create_pushButton">
     <property name="text">
      <string>create gcode</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
